tests/csrf_tests/tests.py

```diff
@@ -203,18 +204,22 @@ def test_process_request_no_csrf_cookie(self):
         Check that if no CSRF cookies is present, the middleware rejects the
         incoming request.  This will stop login CSRF.
         """
-        req = self._get_POST_no_csrf_cookie_request()
-        req2 = CsrfViewMiddleware().process_view(req, post_form_view, (), {})
-        self.assertEqual(403, req2.status_code)
+        with patch_logger('django.security.csrf', 'warning') as logger_calls:
+            req = self._get_POST_no_csrf_cookie_request()
+            req2 = CsrfViewMiddleware().process_view(req, post_form_view, (), {})
+            self.assertEqual(403, req2.status_code)
+            self.assertEqual(logger_calls[0], 'Forbidden (CSRF cookie not set.): ')
```

Comment:
Perhaps you'd like to import the messages from django.middleware.csrf to reduce the repetition and the need to edit all the tests if a message changes.